src/counter.cpp

```diff
@@ -488,6 +488,9 @@ void Counter::one_measurement_count(
     int64_t num_explored = 0;
     int64_t lower_fib = 0;
     int64_t upper_fib = total_max_xors;
+    threshold_sols[total_max_xors] = 0;
```

Comment:
When the sampling set is empty, total_max_xors can be 0, which in turn makes this set the number of solutions to 1. Fuzzer is smart! :)